Entities/AHuman.cpp

```diff
@@ -3129,173 +3129,172 @@ void AHuman::Update()
-    m_Paths[BGROUND][CLIMB].SetHFlip(m_HFlipped);
-    m_Paths[FGROUND][STAND].SetHFlip(m_HFlipped);
-    m_Paths[BGROUND][STAND].SetHFlip(m_HFlipped);
-
-    ////////////////////////////////////
-    // Jetpack activation and blast direction
-
-    if (m_pJetpack && m_pJetpack->IsAttached())
-    {
-        // Start Jetpack burn
-        if (m_Controller.IsState(BODY_JUMPSTART) && m_JetTimeLeft > 0 && m_Status != INACTIVE)
-        {
-            m_pJetpack->TriggerBurst();
-            // This is to make sure se get loose from being stuck
-            m_ForceDeepCheck = true;
-            m_pJetpack->EnableEmission(true);
-            // Quadruple this for the burst
-            m_JetTimeLeft -= g_TimerMan.GetDeltaTimeMS() * 10;
-            if (m_JetTimeLeft < 0)
-                m_JetTimeLeft = 0;
-        }
-        // Jetpack is ordered to be burning, or the pie menu is on and was burning before it went on
-        else if ((m_Controller.IsState(BODY_JUMP) || (m_MoveState == JUMP && m_Controller.IsState(PIE_MENU_ACTIVE))) && m_JetTimeLeft > 0)
-        {
-            m_pJetpack->EnableEmission(true);
-            // Jetpacks are noisy!
-            m_pJetpack->AlarmOnEmit(m_Team);
-            // Deduct from the jetpack time
-            m_JetTimeLeft -= g_TimerMan.GetDeltaTimeMS();
-            m_MoveState = JUMP;
-            m_Paths[FGROUND][JUMP].Restart();
-            m_Paths[BGROUND][JUMP].Restart();
-        }
-        // Jetpack is off/turning off
-        else
-        {
-            m_pJetpack->EnableEmission(false);
-            if (m_MoveState == JUMP)
-                m_MoveState = STAND;
-
-            // Replenish the jetpack time, twice as fast
-            m_JetTimeLeft += g_TimerMan.GetDeltaTimeMS() * 2;
-            if (m_JetTimeLeft >= m_JetTimeTotal)
-                m_JetTimeLeft = m_JetTimeTotal;
-        }
-
-		float maxAngle = c_HalfPI * m_JetAngleRange;
-        // If pie menu is on, keep the angle to what it was before
-        if (m_Controller.IsState(PIE_MENU_ACTIVE))
-        {
-            // Don't change anything
-        }
-        // Direct the jetpack nozzle according to movement stick if analog input is present
-        else if (m_Controller.GetAnalogMove().GetMagnitude() > 0.1)
-        {
-			float minAngle = -maxAngle - c_HalfPI;
-			maxAngle -= c_HalfPI;
-			float jetAngle = std::clamp(m_Controller.GetAnalogMove().GetAbsRadAngle(), minAngle, maxAngle) - c_PI;
-            m_pJetpack->SetEmitAngle(FacingAngle(jetAngle));
-        }
-        // Or just use the aim angle if we're getting digital input
-        else
-        {
-            float jetAngle = m_AimAngle > 0 ? (m_AimAngle * m_JetAngleRange) : 0;
-			jetAngle = jetAngle - maxAngle - c_HalfPI;
-            // Don't need to use FacingAngle on this because it's already applied to the AimAngle since last update.
-            m_pJetpack->SetEmitAngle(jetAngle);
-        }
-    }
+	float deltaTime = g_TimerMan.GetDeltaTimeSecs();
```

Comment:
Is all this bunch of stuff just whitespace, with the exception of the jetpack stuff that's the same as in ACrab?